### PR TITLE
chore: add debugger config for vs code

### PR DESCRIPTION
**Pull request by kappa90**
## Problem

Similar to the debugger configs already existing for PyCharm, I created a launch.json that can be used by VSCode to launch the app in debugger mode.

## Changes

- Removed .vscode from gitignore
- Added a launch.json config file

### CODE REVIEW

**Comment by mariusandra** [comment]
Gonna tag a few more vscode users here. 

Everyone using pycharm has for a while gotten this menu for free:

image

(ignore the broken looking entries... the point is you can click "PostHog" and "debug" and it just works)

All vscode devs have to either run `bin/start`, wait forever, and code... or sacrifice a goat to the terminal.

This PR adds separate launch configurations to vscode, so that if you want, you can start/stop our services (frontend, django, celery, plugin-server) independently of each other.

As a dying breed of non-vscode users, I'd like to ask actual vscode users, if these changes make sense and/or if this would conflict with anything?

**Review by pauldambra — APPROVED** [review]
I'm also not a vscode user...

But, switching to this branch vscode presented me with those configs to run and let me run them

File: .gitignore [hunk 1]
```diff
@@ -13,7 +13,6 @@ debug.log
 node_modules/
 *.code-workspace
 node_modules
-.vscode/
```
Comment (benjackwhite) [inline .gitignore:16]: Do we potentially want to restrict this to just `.vscode/settings.json` as I'm not sure if we necessarily want common settings

**Review by benjackwhite — CHANGES_REQUESTED** [review]
Looks good other than the gitignore change that might be too eager (or at least should be a separate PR to agree on a common `settings.json` if we do want to commit that

File: .gitignore (hunk 1 above)
Comment (kappa90) [reply .gitignore:16]: From my past experience, `.vscode/settings.json` has always been used for shared workspace settings, which overwrites any user configuration. I'm happy to scope down the `.gitignore` to just `launch.json` while keeping `settings.json` customizable, if that's not been used so far.

Comment (benjackwhite) [reply .gitignore:16]: Yeah thats what I was thinking for now. Just ignore everything that **isn't** `launch.json`. At least for this PR. Not against adding a common `settings.json` if we agree on a sensible list though.

Comment (kappa90) [reply .gitignore:16]: Just pushed a change to fix this, thanks!

**Review by Twixes — APPROVED** [review]
Haven't ran this, but makes sense to have a common setup and we can always iterate

**Review by benjackwhite — APPROVED** [review]

**Pull request merged**